vendor.mod

```diff
@@ -62,7 +62,7 @@ require (
 	github.com/miekg/dns v1.1.66
 	github.com/mistifyio/go-zfs/v3 v3.0.1
 	github.com/mitchellh/copystructure v1.2.0
-	github.com/moby/buildkit v0.23.0-rc2
+	github.com/moby/buildkit v0.23.1
```

Comment:
This also includes https://github.com/moby/moby/pull/50222 ? Did we skip that commit?

Reply:
We didn't backport the previous PR, but I don't think it's needed now?

Reply:
Yeah, mostly that we normally use the "cherry-pick -x" so that we have a clear reference to where it was backported from, so in this case, I'd have expected both commits to be found, leading to the original commits/PRs

Reply:
Not sure if there's a real value in recording the intermediate commit, but updated the PR to also include it.